nanvm-lib/src/common/bit_subset64.rs

```diff
@@ -134,12 +130,12 @@ where
         self.superposition() & set
     }
     #[inline(always)]
-    pub const fn from_value(self, value: u64) -> u64 {
+    pub const fn with_value(self, value: u64) -> u64 {
```

Comment:
The function converts/maps a raw value into a subset value (a value inside the subset). Better names:
- `value_to_subset_value`
- `to_subset_value`
- `into_subset_value`

Reply:
You choose! The imp does | over 'self' with 'value', that's why I used 'with_' after looking up Clippy suggestions.

Reply:
`raw_value_to_subset_value`

Reply:
Let me use value_to_subset_value in the next iteration.